README.md

```diff
@@ -32,6 +32,11 @@ DefaultMembers:
         Email: another@example.com
         FirstName: Amazing
         Surname: Example
+  remove:
+    -
+        Email: mundane@example.com
+        FirstName: Mundane
+        Surname: Example
```

Comment:
Maybe this ought to be:
```
delete:
  - mundane@example.com
  - boring@example.com
```
Reasoning:
- `FirstName` and `Surname` are not used by the remove function
- "Remove" can be ambiguous... Is it deleting the member or is it keeping the member, but removing the member's access (e.g. removing all the groups or suspending the user)?

Side note: Maybe we should have a third solution that allows you to keep the member, but suspends them?  (I believe you can suspend a user by setting `LockedOutUntil` to sometime in the year 200017. heh)

Reply:
Sounds like something we can do in the future, we have 198000 years till then.